libraries/aws/shadow/cbmc/proofs/Shadow_MatchTopic/Makefile

```diff
@@ -25,12 +25,17 @@ HARNESS_FILE=Shadow_MatchTopic_harness
 DEFINES +=
 INCLUDES +=
 
-# The thing name field length is bounded.
-MAX_THINGNAME_FIELD_LENGTH=128
+# The thing name field length is bounded because it's maximum length is 64KB which
+# we need to bound it because it's too long. 
+MAX_THINGNAME_FIELD_LENGTH=256
```

Comment:
Please change this to TOPIC_STRING_LENGTH_MAX and move it to right before line 25.
And please update line 25 to:
```
DEFINES += -DTOPIC_STRING_LENGTH_MAX=$(TOPIC_STRING_LENGTH_MAX)
```